dosagelib/plugins/e.py

```diff
@@ -214,6 +214,28 @@ class Evon(WordPressScraper):
     adult = True
 
 
+class ExorcismAcademy(ParserScraper):
+    url = 'https://ea.asmodrawscomics.com/'
+    stripUrl = url + 'comic/%s/'
+    firstStripUrl = stripUrl % 'title-page'
+    imageSearch = '//div[contains(@class, "webcomic-image")]//img[contains(@class, "size-full")]'
+    prevSearch = '//a[contains(@class, "previous-webcomic-link")]'
+    multipleImagesPerStrip = True
+    adult = True
+
+    def namer(self, image_url, page_url):
```

Comment:
Is this just a more fancy version of [joinPathPartsNamer](https://github.com/webcomics/dosage/blob/master/dosagelib/helpers.py#L29)? Wouldn't it make sense to expand that instead?

Reply:
I'm sorry, I haven't figured out how use `joinPathPartsNamer` to zero-pad and pull out elements. Or to split on the preceding "-" (which is not always there).